zng/type.go

```diff
@@ -380,3 +380,14 @@ func ReferencedID(typ Type) int {
 	}
 	return typ.ID()
 }
+
+func TypeRecordOf(typ Type) *TypeRecord {
+	for {
+		alias, ok := typ.(*TypeAlias)
+		if !ok {
+			typ, _ := typ.(*TypeRecord)
+			return typ
+		}
+		typ = alias.Type
+	}
```

Comment:
```suggestion
    typ, _ := zng.AliasOf(typ).(*TypeRecord)
    return typ
```

Reply:
This is called AliasedType.  I like AliasOf better so I will change it throughout on this PR.